spec/index.html

```diff
@@ -248,5 +249,155 @@ <h2>Basic Concepts</h2>
+              representation. The <code>profile</code> parameter MAY be used by
+              clients to express their preferences in the content negotiation process.
+              If the profile parameter is given, a server SHOULD return a document that
+              honors the profiles in the list which it recognizes,
+              and MUST ignore the profiles in the list which it does not recognize.
+              It is RECOMMENDED that profile URIs are dereferenceable and provide
+              useful documentation at that URI. For more information and background
+              please refer to [[RFC6906]].</p>
+            <p>This specification defines seven values for the <code>profile</code> parameter.</p>
+            <dl>
+              <dt><code>http://www.w3.org/ns/json-ld#expanded</code></dt>
+              <dd>To request or specify <a data-cite="json-ld11#dfn-expanded-document-form">expanded YAML-LD document form</a>.</dd>
+              <dt><code>http://www.w3.org/ns/json-ld#compacted</code></dt>
+              <dd>To request or specify <a data-cite="json-ld11#dfn-compacted-document-form">compacted YAML-LD document form</a>.</dd>
+              <dt><code>http://www.w3.org/ns/json-ld#context</code></dt>
+              <dd>To request or specify a <a data-cite="json-ld11#dfn-context-document">YAML-LD context document</a>.</dd>
+              <dt><code>http://www.w3.org/ns/json-ld#flattened</code></dt>
+              <dd>To request or specify <a data-cite="json-ld11#dfn-flattened-document-form">flattened YAML-LD document form</a>.</dd>
+              <dt><code>http://www.w3.org/ns/json-ld#frame</code></dt>
+              <dd>To request or specify a <a data-cite="json-ld11-framing#dfn-frame">YAML-LD frame document</a>.</dd>
+              <dt><code>http://www.w3.org/ns/json-ld#framed</code></dt>
+              <dd>To request or specify <a data-cite="json-ld11#dfn-framed-document-form">framed YAML-LD document form</a>.</dd>
+              <dt><code>http://www.w3.org/ns/json-ld#extended</code></dt>
+              <dd>To request or specify <a data-lt="extended document form">extended YAML-LD document form</a>.
+                <div class="ednote">
+                  This is a placeholder for specifying something like an
+                  <dfn data-lt="extended-document-form">extended YAML-LD document form</dfn>
+                  making use of YAML-specific features.
+                </div></dd>
+            </dl>
+            <p>All other URIs starting with <code>http://www.w3.org/ns/json-ld</code>
+              are reserved for future use by JSON-LD specifications.</p>
+            <!--p>Other specifications MAY create further structured subtypes
+              by using `+ld+json` as a suffix for a new base subtype, as in
+              `application/example+ld+json`.
+              Unless defined otherwise, such subtypes use the same
+              fragment identifier behavior as `application/ld+json`.</p-->
```

Comment:
If we think this is the case, we need to file an issue and remove the commented text from the spec.

Otherwise, remove.

```suggestion
```